src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-list/cephfs-list.component.ts

```diff
@@ -95,6 +98,13 @@ export class CephfsListComponent extends ListWithDetails implements OnInit {
         click: () => this.removeVolumeModal(),
         name: this.actionLabels.REMOVE,
         disable: this.getDisableDesc.bind(this)
+      },
+      {
+        name: this.actionLabels.ATTACH,
+        permission: 'read',
+        icon: Icons.bars,
+        disable: () => !this.selection?.hasSelection,
+        click: () => this.showAttachInfo()
```

Comment:
this should have precedence before `Remove` and after `Edit`. `Remove` should be the final action in the list.

Reply:
done